### PR TITLE
Converted all `player.cardsInHand.push(game.dealer.dealCard())` to `p…

### PR DESCRIPTION
**Pull request by sienmich**
…layer.drawCard(game)`.

This seems simpler to me.

### CODE REVIEW

File: src/cards/promo/HiTechLab.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class HiTechLab implements IProjectCard {
           const cardsDrawn: Array<IProjectCard> = [];
           for (let counter = 0; counter < amount; counter++) {
             cardsDrawn.push(game.dealer.dealCard());
```
Comment (kberg) [inline src/cards/promo/HiTechLab.ts:37]: Did you skip this?

**Review by kberg — COMMENTED** [review]
LG, just checking one thing before approving.

File: src/cards/promo/HiTechLab.ts (hunk 1 above)
Comment (sienmich) [reply src/cards/promo/HiTechLab.ts:37]: Yep, here we select one card to draw from many, so i skipped. However this pattern also repeats many times, so I'll add this as an option. Thanks!

Comment (kberg) [reply src/cards/promo/HiTechLab.ts:37]: Do it separately, this can be pulled in./

**Review by kberg — APPROVED** [review]

File: src/Player.ts [hunk 2]
```diff
@@ -1303,6 +1304,12 @@ export class Player implements ISerializable<SerializedPlayer> {
     );
   }
 
+  public drawCard(game: Game, n = 1): void {
+    range(n).forEach(() => {
```
Comment (Lynesth) [inline src/Player.ts:1308]: Why create an array instead of a simple `for` loop ?

Comment (sienmich) [reply src/Player.ts:1308]: Changed in https://github.com/bafolts/terraforming-mars/pull/2292

**Pull request merged**